web/src/i18n/locales/en.yaml

```diff
@@ -334,8 +334,10 @@
     helpText: "This itemized budget table has combined the costs for each activity into a single itemized table by federal fiscal year (FFY). The table displays total share, federal share, and state share. The second table also includes a proposed breakout of the budget by FFY quarter."
     summaryBudget: 
       title: Summary Budget Table
+      helpText: "Totals are calculated automatically."
     quarterlyBudget: 
       title: Federal Share by FFY Quarter
+      helpText: "Personnel costs for this section includes combined salary and benefits. Salary amount is the yearly cost of the individual personnel listed and personnel only need to be listed once. FTE% is the percentage attributable to the activity. Percentages are automatically calculated from the %FTE provided."
```

Comment:
@quarterback Was this in the wrong place? Looks like this should be at line 235 to address the previous comment?

Reply:
oops